resources/dicts/events/death/forest/general.json

```diff
@@ -82,5 +82,27 @@
             "OMEN,3",
             "SWIMMER,3"
         ]
+    },
+    {
+        "camp": "any",
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare"
+        ],
+        "death_text": "m_c was found dead underneath an oak tree with dog scent wreathing around them, after going on a solo patrol and not coming back.",
```

Comment:
remove mention of a solo patrol, as it's too similar to a patrol that already exist in the game. Recommend instead that m_c was going to the dirtplace or something. I like the. mention of the oak tree, more of our death events need to be biome specific like this

Reply:
alright!